src/main/fc/fc_tasks.c

```diff
@@ -442,11 +442,11 @@ cfTask_t cfTasks[TASK_COUNT] = {
     },
 #endif
 
-#ifdef USE_ESC_TELEMETRY
-    [TASK_ESC_TELEMETRY] = {
-        .taskName = "ESC_TELEMETRY",
-        .taskFunc = taskEscTelemetry,
-        .desiredPeriod = 1000000 / 100,         // 100 Hz
+#ifdef USE_ESC_SENSOR
+    [TASK_ESC_SENSOR] = {
+        .taskName = "ESC_SENSOR",
+        .taskFunc = taskEscSensor,
```

Comment:
OK, i see many wrapper functions, thought this was something common to do with tasks.